webapp-django/crashstats/crashstats/jinja2/crashstats_base.html

```diff
@@ -122,7 +122,7 @@ <h1>Product Navigation</h1>
         |
         <a href="https://bugzilla.mozilla.org/enter_bug.cgi?product=Socorro">File a bug</a>
         |
-        <a href="{{ url('supersearch:search') }}?product={{ product }}{% if version and version != 'ALL' %}&amp;version={{ version }}{% endif %}&amp;_dont_run=1">Super Search</a>
+        <a href="{{ url('supersearch:search') }}?product={% if product %}{{ product.name }}{% endif %}{% if version and version != 'ALL' %}&amp;version={{ version }}{% endif %}&amp;_dont_run=1">Super Search</a>
```

Comment:
This is a messy link. There are some pages on Crash Stats where there's no "product" or "version" in the context. Before, since this was just a variable, it'd be empty because we have undefined variables as empty strings configured in Jinja2. I think that's a gross error and will change it some day.

In the meantime, this continues the original behavior but without the "undefined means empty string" thing.